### ISSUE TITLE
[Documentation] Make Playground Link Noticeable or Change to the Button

### ISSUE THREAD

**Issue by dimitryzub** [serpapi/public-roadmap#329]
I've got feedback from our user about our documentation pages: 

> It would be nice to have a link to the Playground with a noticeable separate button (optional). I know it's there, but I skimmed the entire page until I found it.

## Example 

![image](https://user-images.githubusercontent.com/78694043/189867316-a0286a47-54f2-4c51-b243-b1ea104edb00.png)

## Reasons

Quoting the user:

> Visually through Playground is much faster and more convenient to understand how API works.

Different people perceive information differently. Some by reading/watching docs, some by interactively trying out how API elements (in our case parameters) work. 

@hartator, @aciddjus, @tanys123 curious about your vision 🙏

**Comment by aciddjus** [comment]
I like the idea of adding a more visible button. Our code examples playground button is also hidden in a way:

<img width="542" alt="image" src="https://user-images.githubusercontent.com/6929257/189881463-00054247-6ab8-4191-89f5-935684e6f1ac.png">

**Comment by dimitryzub** [comment]
@aciddjus Yes, agree with that also, thank you 👍 It is somewhat not as expressive in terms of what the button is about. Hope it makes sense 🙂
